pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -711,9 +715,26 @@ func configureVifV6Addresses(p *MasqueradePodInterface, err error) error {
 }
 
 func (p *MasqueradePodInterface) startDynamicIPServers(vmi *v1.VirtualMachineInstance) error {
+	if err := p.startRADaemon(); err != nil {
```

Comment:
Should be done only for dual stack cluster. Isn't it? Maybe similar condition to [what the dhcpv6 server](https://github.com/kubevirt/kubevirt/blob/7c8a44cbfc5ec074a2a93f67f93fbce5063a002f/pkg/virt-launcher/virtwrap/network/common.go#L367) has can be used.